romana-install/config.yml

```diff
@@ -0,0 +1,36 @@
+---
+# Ensure stack nodes are ready for installation
```

Comment:
This is for AWS hosts only, right? Will this also run for Vagrant hosts? If not, maybe a comment?